Update CHANGELOGS (#248)
ROCm/rocprofiler-systems: CHANGELOG.md

Full documentation for ROCm Systems Profiler is available at [https://rocm.docs.amd.com/projects/rocprofiler-systems/en/latest/](https://rocm.docs.amd.com/projects/rocprofiler-systems/en/latest/).

## ROCm Systems Profiler 1.1.0 for ROCm 7.0

### Added

- Profiling and metric collection capabilities for VCN engine activity, JPEG engine activity, and API tracing for rocDecode, rocJPEG and VA-APIs.
- How-to document for VCN and JPEG activity sampling and tracing.

> - Support for tracing in Fortran

Reviewer: Is this for #95 ? Or #185 ?

Author: Both. :) 
I'm open to better phrasing. I know that it's a little vague. 
"Support for MPI API tracing in Fortran"?

Reviewer: The PR #95 fixes generic Fortran tracing hence I got confused if we want to mention that fix or just the MPI Fortran feature.

Author: Updated. Let me know what you think. Is it clearer?

Full documentation for ROCm Systems Profiler is available at [https://rocm.docs.amd.com/projects/rocprofiler-systems/en/latest/](https://rocm.docs.amd.com/projects/rocprofiler-systems/en/latest/).

## ROCm Systems Profiler 1.1.0 for ROCm 7.0

### Added

> - Profiling and metric collection capabilities for VCN engine activity, JPEG engine activity, and API tracing for rocDecode, rocJPEG and VA-APIs.

Reviewer 2: ```suggestion
- Profiling and metric collection capabilities for VCN engine activity, JPEG engine activity, and API tracing for rocDecode, rocJPEG, and VA-APIs.
```

- How-to document for VCN and JPEG activity sampling and tracing.

> - Support for tracing Fortran applications

Reviewer 2: ```suggestion
- Support for tracing Fortran applications.
```

> - Support for tracing MPI API tracing in Fortran

Reviewer 2: ```suggestion
- Support for MPI API tracing in Fortran.
```

### Changed

- Replaced ROCm SMI backend with AMD SMI backend for collecting GPU metrics.

> - Using ROCProfiler SDK to trace RCCL API and collect communication counters

Reviewer 2: ```suggestion
- ROCprofiler-SDK is now used to trace RCCL API and collect communication counters.
```

> - Update Dyninst submodule to v13.0

Reviewer 2: ```suggestion
- Updated the Dyninst submodule to v13.0.
```

- Set the default value of `ROCPROFSYS_SAMPLING_CPUS` to `none`

### Resolved issues

> - Fixed GPU metric collection settings with ROCPROFSYS_AMD_SMI_METRICS.

Reviewer 2: ```suggestion
- Fixed the GPU metric collection settings with `ROCPROFSYS_AMD_SMI_METRICS`.
```

- Fixed a build issue with CMake 4.
- Fixed incorrect kernel names shown for kernel dispatch tracks in Perfetto.
- Fixed formatting of some output logs.

## ROCm Systems Profiler 1.0.2 for ROCm 6.4.2

### Optimized

- Improved readability of the OpenMP target offload traces by showing on a single Perfetto track.

### Resolved issues

> - Fixed the file path to the script that merges Perfetto files from multi-process MPI runs. The script has also been renamed from "merge-multiprocess-output.sh" to "rocprof-sys-merge-output.sh".

Reviewer 2: ```suggestion
- Fixed the file path to the script that merges Perfetto files from multi-process MPI runs. The script has also been renamed from `merge-multiprocess-output.sh` to `rocprof-sys-merge-output.sh`.
```
